internal/polkavm/host_call/refine_functions.go

```diff
@@ -70,3 +71,42 @@ func HistoricalLookup(
 
 	return gas, regs, mem, ctxPair, nil
 }
+
+// Import ΩY(ϱ, ω, µ, (m, e), i)
+func Import(
+	gas Gas,
+	regs Registers,
+	mem Memory,
+	ctxPair RefineContextPair,
+	importedSegments []Segment,
+) (Gas, Registers, Memory, RefineContextPair, error) {
+	if gas < ImportCost {
+		return gas, regs, mem, ctxPair, ErrOutOfGas
+	}
+	gas -= ImportCost
+
+	omega7 := regs[A0]
+	omega8 := regs[A1]
+	omega9 := regs[A2]
```

Comment:
is it possible to give more intuitive names? you can take a look at the accumulation host calls for example

Reply:
done